### PR TITLE
Use ghcr in latest helm

### PR DESCRIPTION
**Pull request by STRRL**
### What problem does this PR solve?

> Problem Summary:
We have moved packaging latest images from self-hosted Jenkins to GitHub CI, and using ghcr.io instead of docker.io, but herm charts still using the legacy one.

### What's changed and how it works?

> What's Changed:
- change images in helm to ghcr.io
- update `install.sh`
- fix one expression bug in `install.sh`

### Related changes

- [ ] Need to update `chaos-mesh/website`
- [ ] Need to update `Chaos Dashboard`
- [x] Need to **cheery-pick to release branches**, need to cherry-pick into `release-2.0`.

And I would create another PR for packaging images with GitHub Action for release-2.0.4 later. :)

### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [x] Manual test (add detailed scripts or steps below)
- [ ] No code

I have manually tested both `helm install` and `install.sh`

Side effects

- [ ] Breaking backward compatibility

### Release note 

```text
Please add a release note.

You can safely ignore this section if you don't think this PR needs a release note.
```

### CODE REVIEW

**Review by iguoyr — APPROVED** [review]
LGTM

File: install.sh [hunk 1]
```diff
@@ -35,7 +35,7 @@ FLAGS:
         --force-local-kube   Force reinstall local Kubernetes cluster if it is already installed
         --force-kubectl      Force reinstall kubectl client if it is already installed
         --force-kind         Force reinstall Kind if it is already installed
-        --docker-mirror      Use docker mirror to pull image, dockerhub.azk8s.cn => docker.io, gcr.azk8s.cn => gcr.io
+        --docker-mirror      Use docker mirror to pull image, dockerhub.azk8s.cn => docker.io, gcr.azk8s.cn => gcr.io; ghcr.io are NOT supported now.
```
Comment (YangKeao) [inline install.sh:38]: BTW, I heard these mirrors are only available only for Azure China network. (But they have nothing to do with this PR.)

**Review by YangKeao — APPROVED** [review]
LGTM

File: install.sh (hunk 1 above)
Comment (STRRL) [reply install.sh:38]: Here is a related isue: https://github.com/chaos-mesh/chaos-mesh/issues/2294

Comment (STRRL) [reply install.sh:38]: It seems the policy of azk8s.cn has been more strict: I can not even pull `dockerhub.azk8s.cn/library/nginx`, server responses 403

**Comment by STRRL** [comment]
> cherry pick to release-2.0 failed

I would cherry-pick manually soon.

**Comment by STRRL** [comment]
> > cherry pick to release-2.0 failed
> 
> I would cherry-pick manually soon.

I changed my mind. Uploading 2.0.4 images to both docker hub and ghcr.io, still using docker hub in helm for a while.

**Pull request merged**